Exercise_B/index.html

```diff
@@ -1,34 +1,65 @@
-
-    <!-- Link and CDN required for bootstrap -->
-
-    <!-- Google Font - Lato -->
-    <link
-      href="https://fonts.googleapis.com/css?family=Lato"
-      rel="stylesheet"
-    />
-    <!-- Custom Stylesheet -->
-    <link rel="stylesheet" href="./assets/style/stylesheet.css" />
-  </head>
-
-  <body>
-    <!--Header (flower shop logo block element)-->
-    <!--Banner Jumbotron of the image-->
-
-    <!--Four containers with img / text-->
-    <!-- 4 containers -->
-
-    <!-- Payments Div -->
-    <!-- Consultation Div -->
-
-    <!-- Delivery Div -->
-
-    <!--Scripts required for bootstrap CDN-->
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <title>Flower Shop BS</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <!--If you want to use bootstrap properly this is vital (helps browser to measure the page and readjust the information accordingly-->
+
+  <!-- Link and CDN required for bootstrap -->
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We" crossorigin="anonymous">
+
+  <!-- Google Font - Lato -->
+  <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet" />
+  <!-- Custom Stylesheet -->
+  <link rel="stylesheet" href="./assets/style/stylesheet.css" />
+</head>
+
+<body>
+  <!--Header (flower shop logo block element)-->
+  <header><img src="./assets/images/logo.png" alt="logo"></header>
+
+  <!--Banner Jumbotron of the image-->
+  <section class="hero">
+    <div class=background-image></div>
+    <div>
+      <h1>Beautiful Flowers</h1>
+      <button>Learn more</button>
+    </div>
+  </section>
+
+
+  <!--Four containers with img / text-->
+
+  <!-- 4 containers -->
+  <!-- Payments Div -->
+  <!-- Consultation Div -->
+  <!-- Delivery Div -->
+
+  <div class="container container-fluid width: 100% ">
+    <div class="row pt-2 ">
+      <div class="col-sm col-md-6 col-lg-3"><img src="assets/images/flower-icon.png">
+        <Column> Flowers for every kind of situation
+      </div>
+      <div class="col-sm col-md-6 col-lg-3"><img src="./assets/images/payment-icon.png" alt="payment">
+        <p>All major payments kinds accepted
+      </div>
+      <div class="col-sm col-md-6 col-lg-3"><img src="./assets/images/phone-icon.png" alt="phone">
+        <p>Free consultation
+      </div>
+      <div class="col-sm col-md-6 col-lg-3"><img src="./assets/images/truck-icon.png" alt="truck">
+        <p>City wide delivery
+      </div>
+    </div>
+  </div>
+
+  </section>
+  <!--Scripts required for bootstrap CDN-->
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p"
+    crossorigin="anonymous"></script>
+</body>
+
+</html>
```

Comment:
For the exercise, the webpage works as intended and it follows the instructions at most. But I would suggest making the icons on the page such as the Flower Icon, Payment Icon, Call Icon, Truck Icon bigger in size and additionally adding the green color background that is required for the website.